source/retryable-reads/tests/changeStreams-client.watch-serverErrors.yml

```diff
@@ -0,0 +1,144 @@
+database_name: &database_name "retryable-reads-tests"
+collection_name: &collection_name "coll"
+
+data:
+    - {_id: 1, x: 11}
+    - {_id: 2, x: 22}
+    - {_id: 3, x: 33}
+    
+minServerVersion: '4.1.6' # 4.2
+
+tests:
+    -
+        description: "client.watch succeeds after InterruptedAtShutdown"
+        clientOptions:
+            retryReads: true
+        failPoint: &failCommand_failPoint
+            configureFailPoint: failCommand
+            mode: { times: 1 }
+            data: { failCommands: [aggregate], errorCode: 11600 }
+        operation: &retryable_operation
+            name: watch
+            object: client
+            result: []
```

Comment:
Since the change stream is tailable, how do you intend to assert the contents of the returned iterator?

I believe the [Change Stream test runner](https://github.com/mongodb/specifications/tree/master/source/change-streams/tests#spec-test-runner) instructs drivers to only assert up to the contents of the array in the test's `operation.result` field. I assume drivers can then abandon iteration on the tailable iterator after that point and pass the test. In this case, there is really nothing to assert so I might suggest omitting `result` entirely.